### PR TITLE
add support for service info

### PR DESCRIPTION
**Pull request by billiford**
Since this is a larger PR (most of it is just payload differences), I would recommend reviewing the `applications.go` file, in particular the functions `makeLoadBalancerServerGroupsMap` and `makeServerGroupLoadBalancersMap`. It should be noted that the `/applications/*/serverGroups` endpoint is now retrieving Kubernetes kind Services in addition to its other four resources. The `/applications/*/loadBalancers` endpoint is now retrieving 5 Kinds: Pods, ReplicaSets, StatefulSets, Services,		and Ingresses. Retrieval of pods seems necessary to build the expected response for the pages.

In summary, this PR:
- Fixes load balancers not being able to be deleted (API change was not setting namespace field correctly)
- Adds service info for ReplicaSets and StatefulSets that are fronted by a service
- Adds the neat load balancer icon to resources in the clusters page
- Shows all resources fronted by a service in the load balancers page

From the Clusters page:
image

From the Load Balancers page:
image

### CODE REVIEW

File: pkg/http/core/applications.go [hunk 1]
```diff
@@ -400,37 +446,155 @@ func ListLoadBalancers(c *gin.Context) {
...
+			state = stateDown
+		}
+
+		or := r.u.GetOwnerReferences()
+		for _, o := range or {
+			i := LoadBalancerInstance{
+				Health: LoadBalancerInstanceHealth{
+					Platform: "platform",
+					// TODO get the container name that is fronted by the service.
+					Source: fmt.Sprintf("Container %s", "TODO"),
```
Comment (dmrogers7) [inline pkg/http/core/applications.go:468]: Is this even used in the UI?

**Review by dmrogers7 — APPROVED** [review]
Looks good!
